package.json

```diff
@@ -29,11 +48,25 @@
     "strapi-plugin-slugify": "^2.3.5",
     "styled-components": "^5.2.1"
   },
+  "devDependencies": {
+    "@eslint/js": "^9.7.0",
+    "@types/eslint__js": "^8.42.3",
+    "@types/jest": "^29.5.12",
+    "@types/koa": "2.13.4",
+    "eslint": "^9.7.0",
+    "jest": "^29.7.0",
+    "prettier": "3.3.3",
+    "sqlite3": "^5.1.7",
+    "supertest": "^7.0.0",
+    "ts-jest": "^29.2.3",
+    "typescript": "^5.5.3",
+    "typescript-eslint": "^7.16.1"
+  },
+  "packageManager": "yarn@4.3.1",
   "engines": {
     "node": ">=20.0.0 <=20.x.x"
   },
   "strapi": {
     "uuid": "679cf8f5-e045-4f0a-afc2-5bc6713ebdf0"
-  },
-  "packageManager": "yarn@4.3.1"
```

Comment:
**question(non-blocking):** Why'd this get removed?  (just curious)

Reply:
It just got moved, we're all good